lua/neo-tree/sources/common/commands.lua

```diff
@@ -343,6 +346,53 @@ M.delete_visual = function(state, selected_nodes, callback)
   fs_actions.delete_nodes(paths_to_delete, callback)
 end
 
+M.preview = function(state)
+  local node = state.tree:get_node()
```

Comment:
Since this is being used as the event handler, I would short circuit here if the current window is not the same as `state.winid` to avoid unnecessary processing or unexpected behavior.

Reply:
Good note. I think its best to add the check to the event handler itself since this can also be used as a regular command.